### ISSUE TITLE
What is the approximate range of the best training epoch

### ISSUE THREAD

**Issue by weitaolin999** [wurenkai/H-vmunet#8]
What is the approximate range of the best training epoch

**Comment by wurenkai** [comment]
Hi, in our setup of learning rate and learning strategy, we think 250 or 300 training epochs is a better choice.
